.github/workflows/index-monitor.yml

```
@@ -113,8 +113,9 @@ jobs:
     name: Send notification
     needs: [validation-check, integrity-check]
     if: ${{ always() && (needs.validation-check.outputs.result != 'ok' || needs.integrity-check.outputs.result != 'ok') }}
-    uses: bitnami/charts/.github/workflows/gchat-notification.yml@main
+    uses: bitnami/support/.github/workflows/gchat-notification.yml@main
     with:
       workflow: ${{ github.workflow }}
       job-url: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
-    secrets: inherit
+    secrets:
+      webhook-url: ${{ secrets.GCHAT_WEBHOOK_URL }}
```

Shouldn't be `GCHAT_CONTENT_ALERTS_WEBHOOK_URL`?

```suggestion
      webhook-url: ${{ secrets.GCHAT_CONTENT_ALERTS_WEBHOOK_URL }}
```

---

These alerts go to a different channel